cli/src/main.mjs

```diff
@@ -0,0 +1,266 @@
+
+    // Print summary using stats from processor
+    const totalProcessed = stats.successful + stats.noDocuments + stats.failed;
+
+    console.log(`\n📊 Summary:`);
+    console.log(`  Total processed: ${totalProcessed}`);
+    console.log(`  Successfully scanned: ${stats.successful}`);
+    if (stats.noDocuments > 0) {
+      console.log(`  No documents found: ${stats.noDocuments}`);
+    }
+    console.log(`  Failed: ${stats.failed}`);
+
+    console.log(`\n🎉 Processing completed successfully!`);
+  } catch (error) {
+    console.error(`\n❌ Error: ${error.message}`);
+    process.exit(1);
+  }
+}
+
+/**
+ * Run in interactive mode with prompts
+ */
+async function runInteractiveMode() {
+  try {
+    // 1. Get operation mode
+    const mode = await promptOperationMode();
+
+    let gemiIds = [];
+
+    // 2. Get input based on selected mode
+    switch (mode) {
+      case "file": {
+        const filePath = await promptFileInput();
+        gemiIds = await loadInputFile(filePath);
+        console.log(`✅ Loaded ${gemiIds.length} GEMI ID(s) from file.`);
+        break;
+      }
+
+      case "manual": {
+        gemiIds = await promptManualGemiIds();
+        console.log(`✅ Received ${gemiIds.length} GEMI ID(s).`);
+        break;
+      }
+
+      case "random": {
+        const count = await promptRandomCount();
+        console.log(`🎲 Getting ${count} random companies...`);
+        gemiIds = await getRandomCompanies(count);
+        if (gemiIds.length === 0) {
+          console.log(
+            "❌ Could not find any random companies. Please try again later."
+          );
+          process.exit(0);
+        }
+        break;
+      }
+
+      default:
+        console.log("❌ Invalid operation mode selected.");
+        process.exit(1);
+    }
+
+    // 3. Confirm before processing
+    const confirmed = await promptConfirmation(gemiIds, mode);
+    if (!confirmed) {
+      console.log("Operation cancelled.");
+      process.exit(0);
+    }
+
+    // 4. Process the companies
+    console.log(`\n🚀 Starting processing of ${gemiIds.length} companies...\n`);
+
+    const outputRoot = path.join(projectRoot, "output");
+    const result = await processCompanies(gemiIds, outputRoot);
+    const companies = result.companies;
+    const stats = result.stats;
+
+    // 5. Write output and show summary
+    const outputFile = path.join(outputRoot, "govdoc-output.json");
+    await writeOutput(companies, outputFile);
+
+    // Print summary using stats from processor
+    const totalProcessed = stats.successful + stats.noDocuments + stats.failed;
+
+    console.log(`\n📊 Summary:`);
+    console.log(`  Total processed: ${totalProcessed}`);
+    console.log(`  Successfully scanned: ${stats.successful}`);
+    if (stats.noDocuments > 0) {
+      console.log(`  No documents found: ${stats.noDocuments}`);
+    }
+    console.log(`  Failed: ${stats.failed}`);
+
+    console.log(`\n🎉 Processing completed successfully!`);
```

Comment:
The same applies here for the multiple console.log messages, that you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). They support multiline text, plus they allow you to inject variables using `placeholders` from your code to use in operations using the `${}` syntax.